lib/snapshot-manager.js

```diff
@@ -104,13 +104,32 @@ function combineEntries(entries) {
 	const buffers = [];
 	let byteLength = 0;
 
-	const sortedKeys = [...entries.keys()].sort();
+	const sortedKeys = [...entries.keys()].sort((kA, kB) => {
```

Comment:
```suggestion
	const sortedKeys = [...entries.keys()].sort((keyA, keyB) => {
```

Try to use descriptive variable names. No abbreviations.